moesifasgi/update_companies.py

```diff
@@ -11,91 +14,92 @@ def __init__(self):
                             if 401 <= inst.response_code <= 403:
-                                print("Unauthorized access sending event to Moesif. Please check your Appplication Id.")
+                                logger.error("Unauthorized access sending event to Moesif. Please check your Appplication Id.")
                             if DEBUG:
-                                print("Error while updating company, with status code:")
-                                print(inst.response_code)
+                                logger.info(f"Error while updating company, with status code: {str(inst.response_code)}")
                     else:
-                        print('To update a company, an company_id field is required')
+                        if DEBUG:
+                            logger.info('To update a company, an company_id field is required')
                 except:
-                    print('Error while deserializing the json, please make sure the json is valid')
+                    logger.warning('Error while deserializing the json, please make sure the json is valid')
 
     @classmethod
     def update_companies_batch(cls, company_profiles, api_client, DEBUG):
         if not company_profiles:
-            print('Expecting the input to be either of the type - List of CompanyModel, dict or json while updating companies')
+            logger.info('Expecting the input to be either of the type - List of CompanyModel, dict or json while updating companies')
         else:
             if all(isinstance(company, dict) for company in company_profiles):
                 if all('company_id' in company for company in company_profiles):
                     try:
                         batch_profiles = [CompanyModel.from_dictionary(d) for d in company_profiles]
                         api_client.update_companies_batch(batch_profiles)
                         if DEBUG:
-                            print('Company Profile updated successfully')
+                            logger.info('Company Profile updated successfully')
                     except APIException as inst:
                         if 401 <= inst.response_code <= 403:
-                            print("Unauthorized access sending event to Moesif. Please check your Appplication Id.")
+                            logger.error("Unauthorized access sending event to Moesif. Please check your Appplication Id.")
                         if DEBUG:
-                            print("Error while updating companies, with status code:")
-                            print(inst.response_code)
+                            logger.info("Error while updating companies, with status code: {str(inst.response_code)}")
                 else:
-                    print('To update companies, an company_id field is required')
+                    if DEBUG:
+                        logger.info('To update companies, an company_id field is required')
```

Comment:
this is warning or error and don't need `debug` here.

Reply:
changed